configs/_base_/datasets/waymoD5-mv3d-3class.py

```diff
@@ -1,5 +1,5 @@
 # dataset settings
-# D3 in the config name means the whole dataset is divided into 3 folds
+# D5 in the config name means the whole dataset is divided into 3 folds
```

Comment:
3 folds or 5 folds?

Reply:
5 folds, old comment is not correct.

Reply:
so we should update the comments to correct them.